eng/Versions.props

```diff
@@ -103,7 +103,7 @@
     <SystemRuntimeInteropServicesVersion>4.3.0</SystemRuntimeInteropServicesVersion>
     <SystemRuntimeInteropServicesRuntimeInformationVersion>4.3.0</SystemRuntimeInteropServicesRuntimeInformationVersion>
     <SystemRuntimeSerializationPrimitivesVersion>4.3.0</SystemRuntimeSerializationPrimitivesVersion>
-    <SystemSecurityAccessControlVersion>5.0.0</SystemSecurityAccessControlVersion>
+    <SystemSecurityAccessControlVersion>6.0.0-rc.2.21454.1</SystemSecurityAccessControlVersion>
```

Comment:
This is OK for now, but we should make sure to remember to update to 6.0.0 stable when available.

Reply:
One other place that might be interesting to pick up 6.0.0 prerelease is here: https://github.com/dotnet/runtime/blob/06b9703f229bc34a81e7de5743fc492d8dda9f34/src/libraries/shims/ApiCompat.proj#L8